src/components/scenes/CreateWalletSelectCryptoScene.tsx

```diff
@@ -288,7 +288,7 @@ const CreateWalletSelectCryptoComponent = (props: Props) => {
     () => (
       <Fade noFadeIn={defaultSelection.length > 0} visible={selectedItems.size > 0} duration={300}>
         <View style={styles.bottomButton}>
-          <ButtonUi4 label={lstrings.string_next_capitalized} onPress={handleNextPress} />
+          <ButtonUi4 label={lstrings.string_next_capitalized} testID="createWalletCryptoNext" onPress={handleNextPress} />
```

Comment:
Did you test that this works? Usually you need to force the parent container to not takeover taps by putting
`accessible={false}` on the parent.

Or the problem is due to absolute positioning.

Reply:
`accessible={false}` had no impact to the behavior. There was no Maestro tappability issues through this View when disabling the fade animation for Maestro builds.

Kind of related: `isMaestro()` itself is broken - does not return true for Maestro builds.

Reply:
Use of `EdgeAnim` helps, but only to a degree. Nesting buttons within a hard-coded styled `View` as done before doesn't work no matter what. `ButtonsView` flavors such as `SceneButtons` works perfectly and makes the spacing consistent.